src/MCPClient/requirements/base.txt

```diff
@@ -7,5 +7,6 @@ django-extensions==1.1.1
 mysqlclient==1.3.7
 gearman==2.0.2
 lxml==3.5.0
+-e git://github.com/artefactual-labs/mets-reader-writer.git@dev/reingest#egg=metsrw
```

Comment:
Fixed - now that dev/reingest has been merged it just checks out mets-reader-writer.